fontforge/python.c

```diff
@@ -856,6 +856,56 @@ static PyObject *PyFF_UnicodeNamesListVersion(PyObject *UNUSED(self), PyObject *
     return( ret );
 }
 
+/* Names2 lookup commands to get info from attached libuninameslist >= v0.5. */
+/* Cnt returns lookup table-size, Nxt returns next Unicode value from array, */
+/* Nxt returns 'n' for table array[0..n..(Cnt-1)] pointer. Errors return -1. */
+static PyObject *PyFF_UnicodeNames2GetCntFromLib(PyObject *UNUSED(self), PyObject *UNUSED(args)) {
+    return( Py_BuildValue("i", unicode_names2cnt()) );
+}
+
+static PyObject *PyFF_UnicodeNames2GetNxtFromLib(PyObject *UNUSED(self), PyObject *args) {
+    long val;
+
+    if ( !PyArg_ParseTuple(args,"|i",&val) )
+	return( NULL );
+    return( Py_BuildValue("i", unicode_names2getUtabLoc(val)) );
+}
+
+static PyObject *PyFF_UnicodeNames2NxtUniFromLib(PyObject *UNUSED(self), PyObject *args) {
+    long val;
+
+    if ( !PyArg_ParseTuple(args,"|i",&val) )
+	return( NULL );
+    return( Py_BuildValue("i", unicode_names2getUtabLoc(val)) );
+}
+
+static PyObject *PyFF_UnicodeNames2FrmTabFromLib(PyObject *UNUSED(self), PyObject *args) {
+    long val;
+    char *temp;
+
+    if ( !PyArg_ParseTuple(args,"|i",&val) )
+	return( NULL );
+    if ( (temp=unicode_name2FrmTab(val))==NULL ) {
+	temp=malloc(1*sizeof(char)); *temp='\0';
```

Comment:
Not a big fan of this. Surely you could just

```c
return Py_BuildValue("s", "");
```

Reply:
Nice replacement. There's a lot of places to replace stuff like this in python.c.